dbms/src/Common/FieldVisitors.h

```diff
@@ -372,6 +389,8 @@ class FieldVisitorAccurateLess : public StaticVisitor<bool>
             return l < r;
         else if constexpr (std::is_same_v<U, Int64> || std::is_same_v<U, UInt64>)
             return l < DecimalField<Decimal128>(r, 0);
+        if constexpr (std::is_same_v<T, Null>)
```

Comment:
It looks wrong.

Reply:
:( You are right, I misinterpreted the meaning of T.